### DISCUSSION TITLE
Nouveau mini event : Lorella

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#1686] [Nouvelles idées]
Lorella est une bibliothécaire qui se balade ici et là à travers le royaume (littéralement, genre elle se déplace entre les lieux aléatoirement à la durée indiqué). Elle adore lire tout ce qu'elle trouve, en particulier les encyclopédies.

Lors d'une balade, si elle tombe sur vous, elle vous contera l'histoire d'un objet en une ou deux phrases (et peut être même vous indiquer ceux qui pourraient être utile pour des quêtes / leur nombre en circulation, etc.)

**Comment by Nargacaura** [comment]
Ça ne me semble pas mal comme idée, mais faudra quand même rajouter une description pour la centaine d'objets s'il en faut une x)

**Reply by romain22222** [reply]
rhooo je pense que ca amusera beaucoup les testeurs / rédacteurs à faire ca ^^

**Reply by BastLast** [reply]
Ça fait un taf de ouf...

**Reply by romain22222** [reply]
ouais mais imagine juste le lore que ca pourrait apporter, ce serait insane !

**Reply by romain22222** [reply]
autre chose, les descriptions ne seraient pas toutes la dès le début, si vraiment y'a trop de taf, on mettrait dans le code qu'on peut tomber uniquement sur les objets qui ont une description

**Reply by romain22222** [reply]
les autres seraient inconnus des encyclopédies :wink:

**Reply by leuhvraiminoryo** [reply]
je suis totalement pour :slightly_smiling_face:

**Comment by BastLast** [comment]
ça serait tellement un bon event pour une api chat gpt
